lib/manageiq/appliance_console/utilities.rb

```diff
@@ -16,6 +16,16 @@ def self.rake_run(task, params)
       result
     end
 
+    def self.rake_run!(task, params)
+      result = run_rake(task, params)
```

Comment:
should `rake_run` method be getting called here?

Reply:
Yes just a little dyslexic this morning :laughing: